Sources/StreamChatUI/Components.swift

```diff
@@ -10,8 +10,8 @@ public typealias Components = _Components<NoExtraData>
 
 /// An object containing types of UI Components that are used through the UI SDK.
 public struct _Components<ExtraData: ExtraDataTypes> {
-    /// The view used as a navigation bar title view for some view controllers.
-    public var navigationTitleView: TitleContainerView.Type = TitleContainerView.self
+    /// A view that displays a title label and subtitle in a container stack view.
+    public var titleContainerView: TitleContainerView.Type = TitleContainerView.self
```

Comment:
we should have two vars so that you can pick different titles for thread and channel screens 

what about:

```swift
class TitleContainerView: _View, ... {}

class ChatChannelHeaderView:  TitleContainerView {}

class ChatThreadHeaderView:  TitleContainerView {}


    public var chatChannelHeaderView: ChatChannelHeaderView.Type = ChatChannelHeaderView.self

    public var chatThreadHeaderView: ChatThreadHeaderView.Type = ChatThreadHeaderView.self


```